### DISCUSSION TITLE
Boost Info / Potion Critique

### DISCUSSION THREAD

**Discussion by YoungNubby** [runelite/runelite#17542] [Ideas]
(<_<) While I am here... I am using the Boost Info to watch my potion stats. I am using the display panel with a buff change timer. 

My first critique is that I do not like that the timer only comes on after the first level. The timer should start right when I take the potion. I want to know how long I have to catch that dragon impling before my level leaves.

/tldr that is it...

Here I am checking my facts to present my case, and I notice the boost timer does restart when I re-dose. Im thinking this is a bug. I only get 2 seconds of a boost if that is what the timer is at? Shouldn't it restart when I redose? I am thinking this is a bug. I go over to the classic client. It is the same thing! The potted buff countdown timer is universal... It does not restart with dose. 

*Mind blown* I can't be the only one who didn't know this going about wasting potions. Make sure to redose the second your _(next)_ level drops to get the most of your potions.

**Comment by Nightfirecat** [comment]
We can't infer how long a boost will last until we see when it debuffs for the first time.
